src/frontend/src/btc/components/transactions/BtcTransactionModal.svelte

```diff
@@ -58,9 +59,21 @@
 	sendToLabel={$i18n.transaction.text.to}
 	typeLabel={type === 'send' ? $i18n.send.text.send : $i18n.receive.text.receive}
 >
-	<!--	TODO: Implement BtcTransactionStatus component	-->
+	{#if nonNullish(confirmations)}
+		<Value ref="confirmations" slot="confirmations">
+			<svelte:fragment slot="label">{$i18n.transaction.text.confirmations}</svelte:fragment>
+			{confirmations}
+		</Value>
+	{/if}
+
 	<Value ref="status" slot="transaction-status">
 		<svelte:fragment slot="label">{$i18n.transaction.text.status}</svelte:fragment>
-		{`${status === 'pending' ? $i18n.transaction.text.pending : 'Confirmed'}`}
+		{`${
+			status === 'pending'
```

Comment:
It's me or we already had this piece of code? Worth extracting a component or funtion for it?

Reply:
We have a similar component that only displays either "Unconfirmed" or "Pending" (`TransactionStatus`), plus adds additional styling to the label (which should not be applied in `BtcTransactionModal`). In case of the modal, we also want to display the third option ("Confirmed"). 

Having said that, I think for now we can keep it as it is since I can't foresee another use-case where all 3 labels are needed. But I'm also fine with creating a component. WDYT?

Reply:
Maybe not in the PR but, yeah it feels like it would deserve a bit of cleanup, agree?

Reply:
Sounds good, agreed.